crates/storage/libmdbx-rs/src/txn_manager.rs

```diff
@@ -347,7 +385,7 @@ mod read_transactions {
                 drop(tx);
 
                 assert!(!read_transactions.active.contains_key(&tx_ptr));
-                assert!(!read_transactions.aborted.contains(&tx_ptr));
+                assert!(read_transactions.aborted.contains(&tx_ptr));
```

Comment:
this is incorrect, `aborted` list should only contain the transactions aborted due to a timeout. Here, it's aborted by the user.